apps/hermes/server/src/api/types.rs

```diff
@@ -244,6 +246,50 @@ impl From<PriceFeedUpdate> for ParsedPriceUpdate {
         }
     }
 }
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct ParsedPriceFeedTwap {
+    pub id: RpcPriceIdentifier,
+    /// The start unix timestamp of the window
+    pub start_timestamp: i64,
+    /// The end unix timestamp of the window
+    pub end_timestamp: i64,
+    /// The calculated time weighted average price over the window
+    pub twap: RpcPrice,
+    /// The % of slots where the network was down over the TWAP window.
+    /// A value of zero indicates no slots were missed over the window, and
+    /// a value of one indicates that every slot was missed over the window.
+    /// This is a float value stored as a string to avoid precision loss.
+    #[serde(with = "pyth_sdk::utils::as_string")]
+    #[schema(value_type = String, example="0.00001")]
+    pub down_slots_ratio: f64,
```

Comment:
Can we consider Decimal from [`rust_decimal`](https://crates.io/crates/rust_decimal) for this field to make it more accurate in calculation as well.

Reply:
good shout, updated!